docs/resources/environment.md

```diff
@@ -138,9 +138,11 @@ resource "rafay_environment" "eks-rds-env-example" {
 
 ***Optional***
 
+- `display_overridden` (Boolean) Display overridden values on UI
+- `restricted_key_values` (Map of String) If the override type is restricted_key_values, values it is restricted to
 - `restricted_values` (List of String) If the override type is restricted, values it is restricted to
-- `type` (String) Specify the type of override this variable supports, Available types are `allowed`, `notallowed`, `restricted`
```

Comment:
Need to specify the types we support - Available types are `allowed`, `notallowed`, `restricted`